megalut/learn/tenbilacrun.py

```diff
@@ -113,7 +113,7 @@ def predict(catalog, conflist, workbasedir=None):
 		if os.path.isfile(toolconfpath):
 			toolconfig = readconfig(toolconfpath)
 			trainworkdir = os.path.join(workbasedir, dataconfig.get("setup", "name") + "_" + toolconfig.get("setup", "name"))
-		elif os.path.isdir(toolconfpath)
+		elif os.path.isdir(toolconfpath):
```

Comment:
This was fixed already, I think -- maybe remerge the latest master into the branch ?